restapi/src/main/java/io/stargate/web/RestApiActivator.java

```diff
@@ -75,6 +77,21 @@ public void start(BundleContext context) throws InvalidSyntaxException {
         }
       }
 
+      refs = context.getServiceReferences(AuthorizationService.class.getName(), null);
+      if (refs != null) {
+        for (ServiceReference ref : refs) {
+          // Get the service object.
+          Object service = context.getService(ref);
+          if (service instanceof AuthorizationService
+              && ref.getProperty("AuthIdentifier") != null
+              && ref.getProperty("AuthIdentifier").equals(AUTH_IDENTIFIER)) {
```

Comment:
nit: I believe we can pass this as a filter to `getServiceReferences`

Reply:
It also looks like a good use case for the "base activator" class.

Reply:
I believe the idea is to come back and refactor our activators with that.

Reply:
:+1: